sites/public/styles/overrides.scss

```diff
@@ -165,6 +165,32 @@ li.progress-nav__item {
   width: 20%;
 }
 
+.progress-nav__item {
+  &.is-active:before {
+    @apply bg-primary-dark;
+  }
+  &.is-disabled:before {
+    @apply bg-white;
+    @apply border-2;
+    @apply border-primary;
+  }
+  &:before {
+    background-color: #019682;
```

Comment:
Is this color correct? It seems like it's very similar to the primary color?

Reply:
I copy-pasted this from _somewhere_, but for the life of me I can't find it in the Figma mocks anymore... :thinking: